package.json

```
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 8080",
```

the backend configured it so that we use port 3001
```suggestion
    "dev": "next dev -p 3001",
```

---

Agree